statistics/handle/cache/statscacheinner.go

```diff
@@ -118,6 +118,7 @@ func (sc *StatsCache) Put(id int64, t *statistics.Table) {
 }
 
 func (sc *StatsCache) putCache(id int64, t *statistics.Table) bool {
+	metrics.UpdateCounter.Inc()
```

Comment:
Should we add these metrics on `LFU` instead of `StatsCache`?

Reply:
In fact, MapCache will use it too.

Reply:
In the future, Every cache implementation will have same metrics.